src/discordBot.py

```diff
@@ -253,6 +253,28 @@ async def sendDailyRecap():
     recap = await dedicated_channel.send(embed=embed)
     await recap.pin()
 
+aiocron.crontab('59 7 * * *')
+async def send_top16_daily_recap():
+    top16_players_in_each_region = leaderboardBot.get_leaderboard_range(1,16)
+
+    discord_payload = ""
+
+    def append_to_discord_payload(string):
+        discord_payload += string
+        discord_payload += "\n"
+    for region in top16_players_in_each_region.keys():
+        append_to_discord_payload(f"**{region} Top 16 **")
+        for rank,player in top16_players_in_each_region[region]:
```

Comment:
Rank is actually rating, rank isn't stored in the return of top16_players but the dictionary is in reverse order where the first key to iterate on is the 16th player or whatever the end range # is.

Reply:
Addressed.

Reply:
I think it would be useful to add an abstraction that translates serialized database data and deserialized objects.

That way humans can interact with documented classes with helper methods, and machines (dynamodb) can ingest the raw json it desires. I have some examples I could show you in current projects.